pkg/metrics/metrics.go

```diff
@@ -123,14 +123,13 @@ func postRequest(output []byte) error {
 	if resp.StatusCode != http.StatusOK {
 		body, err := ioutil.ReadAll(resp.Body)
 		if err != nil {
+			fmt.Errorf("HTTP error code: %d", resp.StatusCode)
```

Comment:
`fmt.Errorf` creates an error instance we are not using.

Reply:
feel free to use log.Printf here as these are guarded by `err != nil` and won't be printed everytime.